string planner_name
socialrobot_behavior/PlannerInputs inputs
socialrobot_msgs/Behavior requirements
---
bool result
socialrobot_behavior/PlannerOutputs motion
